CHANGELOG.md

```diff
@@ -41,6 +38,7 @@
 - Make sure to rename the API query parameter `layerName` to `layerKey` and API endpoint `listLayerNames` to `listLayerKeys` ([#376])
 - To continue to retrieve the properties of the GeoJSON API response as flat list, you need to set the API request parameter `flattem` to `True` ([#375])
```

Comment:
Maybe replace the 'flattem' with 'flatten' while you are at it anyway

Reply:
Done